### DISCUSSION TITLE
Delays and inconsistencies

### DISCUSSION THREAD

**Discussion by Brian-1150** [planetscale/discussion#515] [Bug Reports]
This has happened numerous times while testing. I insert a known number of records from my app. In this latest example it was exactly 272. Immediately after the insert, I try a
"SELECT COUNT(*)" and see only 256. I try a few times. Still 256. I go back to my app and look and can't seem to find 16 missing items. I come back to PlanetScale and run another "SELECT COUNT(*)" and all 272 are there.
So I go to insights to see if maybe the insert just took a really long time. Nope, less than 3000ms. However, it claims that only 271 records were inserted. What might be going on here?

**Comment by mscoutermarsh** [comment]
Hi @Brian-1150,

How are you running the queries? Are they all coming from your app? Can you share more about the app?

This sounds to me like there may be some caching happening somewhere in your application. PlanetScale would not return a stale result unless you are querying against a replica (sounds like you are not) and there is considerable replication lag (database is under high load, which also sounds unlikely)
